src/catalog/src/system_schema/information_schema/tables.rs

```diff
@@ -259,7 +260,13 @@ impl InformationSchemaTablesBuilder {
                 let table_info = table.table_info();
 
                 // TODO(dennis): make it working for metric engine
-                let table_region_stats = if table_info.meta.engine == MITO_ENGINE {
+                let table_region_stats = if table_info.meta.engine == MITO_ENGINE
+                    || table_info
+                        .meta
+                        .options
+                        .extra_options
+                        .contains_key(PHYSICAL_TABLE_METADATA_KEY)
```

Comment:
Can we add a method to `TableInfo` to check if a table is a "physical table"?

Reply:
Refactor at https://github.com/GreptimeTeam/greptimedb/pull/4975/commits/9bcf284bccba200bd56946a1214b418809573e0b